yanglint.mk

```
@@ -0,0 +1,16 @@
+
+CHECK_TARGETS = $(addsuffix .checkyang,$(drafts_source))
+.PHONY: yanglint $(CHECK_TARGETS)
+
+$(CHECK_TARGETS): %.checkyang: %
+	@if grep -q -E '^\s*YANG-(MODULE|DATA|TREE)' $< ; then \
+	  if ! $(LIBDIR)/yang-check.sh $< ; then \
+	    echo "$(LIBDIR)/yang-check.sh $< failed" ; \
+	    exit 1 ; \
+	  fi ; \
+	else \
+		echo "(no yang imported in $<)" ; \
+	fi
+
+yanglint: $(CHECK_TARGETS)
+
```

```suggestion
2019-10-30 12#244 
```
[RFC 2119 - Key.txt](https://github.com/martinthomson/i-d-template/files/5325359/RFC.2119.-.Key.txt)

```

---

```suggestion
2019-10-30 12#244 
```